content/06.model-complexity.md

```diff
@@ -11,30 +11,31 @@ All of these contribute to low signal to noise ratio in rare disease datasets.
 Additional approaches like combining random forests with resampling without replacement can generate confidence intervals for the model predictions by iteratively exposing the models to incomplete datasets, mimicking real world cases where most rare disease datasets are incomplete [@doi:10.3390/genes11020226].
 Resampling approaches are most helpful in constructing confidence intervals for algorithms that generate the same outcome every time they are run (i.e., deterministic models).
 For decision trees that choose features at random for selecting a path to the outcome (i.e., are non-deterministic), resampling approaches can be helpful in estimating the reproducibility of the model. 
 
-In situations where decision tree-based ensemble methods fail when they are applied to rare disease datasets, cascade learning has been a viable alternative [@doi:10.1007/s11634-019-00354-x; @pmid:30815073].
+In situations where decision tree-based ensemble methods fail when they are applied to rare disease datasets, cascade learning has been a viable alternative [@doi:10.1007/s11634-019-00354-x; @pmid:30815073]. [TODO: first reference is generally about decision trees in imbalanced situations, not about cascade learning per se -- consider moving elsewhere in the sentence?]
 In cascade learning, multiple methods leveraging distinct underlying assumptions are used in tandem to capture stable patterns existing in the dataset [@doi:10.1109/cvpr.2001.990537; @doi:10.1007/978-3-540-75175-5_16; @doi:10.1109/icpr.2004.1334680]. 
 For example, a cascade learning approach for identifying rare disease patients from electronic health record data incorporated independent steps for feature extraction (word2vec [@arxiv:1301.3781]), preliminary prediction with ensembled decision trees, and then prediction refinement using data similarity metrics [@pmid:30815073]. 
-Combining these three methods resulted in better overall prediction when implemented on a silver standard dataset, as compared to using the methods in isolation.
-In addition to cascade learning, other approaches that can better represent rare classes like inverse class weighting and oversampling [doi:10.1613/jair.953] may also help mitigate limitations due to class imbalance. 
+Combining these three methods resulted in better overall prediction when implemented on a silver standard dataset, as compared to a model that used ensemble-based prediction alone. [TODO: my understanding of the baseline model is that there was no feature selection or prediction refinement, but that prediction refinement did not get tested alone as the earlier language implied -- check my understanding]
+In addition to cascade learning, other approaches that can better represent rare classes like inverse class weighting [TODO: Add ref?] and oversampling [doi:10.1613/jair.953] may also help mitigate limitations due to class imbalance. 
```

Comment:
Should we add a reference for inverse class weighting specifically?

Reply:
Noting that I believe addressing these comments about (what we are currently calling) inverse class weighting is the last thing we wanted to do for _this pull request_. My understanding is that @jaybee84 will make suggestions here on this branch/PR.

Reply:
```suggestion
In addition to cascade learning, other approaches that can better represent rare classes using class re-balancing techniques like inverse sampling probability weighting [@doi:10.1186/s12911-021-01688-3], inverse class frequency weighting [@doi:10.1197/jamia.M3095], oversampling of rare classes [doi:10.1613/jair.953], or undersampling of majority class [@doi:10.48550/arXiv.1608.06048] may also help mitigate limitations due to class imbalance. 
```